assets/stylesheets/host.css

```diff
@@ -9,4 +10,231 @@ body {
 .frame {
   height: 100%;
   width:  100%;
+  margin-top: 15px;
+}
+
+.modal-container {
+  animation: modalContainerFade ease 0.5s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: none;
```

Comment:
`display: none;` is not inherent to this CSS class.  There is a separate class for visibility that is defined in `main.css`: `.hidden`.  You can toggle that class on an element to toggle its visibility.

Reply:
Good call - done! `toggle(..)` ftw!